tests/common/devices.py

```diff
@@ -20,6 +20,20 @@
 from errors import UnsupportedAnsibleModule
 
 
+# This is a hack for issue https://github.com/Azure/sonic-mgmt/issues/1941
+# Detailed root cause analysis of the issue: https://github.com/Azure/sonic-mgmt/issues/1941#issuecomment-670434790
+# Before calling callback function of plugins to return ansible module result, ansible calls the
+# ansible.executor.task_result.TaskResult.clean_copy method to remove some keys like 'failed' and 'skipped' in the
+# result dict. The keys to be removed are defined in module variable ansible.executor.task_result._IGNORE. The trick
+# of this hack is to override this pre-defined key list. When the 'failed' key is not included in the list, ansible
+# will not remove it before returning the ansible result to plugins (pytest_ansible in our case)
+try:
+    from ansible.executor import task_result
+    task_result._IGNORE = ('skipped')
```

Comment:
`_IGNORE` should be `('skipped',)`.

Reply:
Yes, in the original ansible code, `_IGNORE` is a tuple with two items: `("failed", "skipped")`. Here we override it with a new tuple to exclude the `failed` item.

Reply:
`('skipped')` is a string, should be `('skipped',)`

Reply:
Good catch!